_data/simple-icons.json

```diff
@@ -11622,6 +11622,17 @@
             "title": "Zyte",
             "hex": "B02CCE",
             "source": "https://www.zyte.com/"
+        },
+        {
+            "title": "Asus ROG",
```

Comment:
This data is in alphabetical order

Reply:
I have arranged the entire data in alphabetical order, next to ASUS data.